Backtracking/Partition Equal Subset Sum.py

```diff
@@ -0,0 +1,55 @@
+`___________________________________________________________________GFG-PROBLEM_______________________________________________________________________________________`
```

Comment:
It is not necessary to specify that. Please remove this first line and the last line. One more thing, use underscores while renaming the files instead of spaces. For example, change it to partition_equal_sum.py